SUIApp/SUIApp/BasketView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BasketView.swift
+//  SUIApp
+//
+//  Created by Vladka on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Экран корзины
+struct BasketView: View {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let currentTariffText = "Текущий тариф"
+        static let nameTitleText = "Название:"
+        static let periodTitleText = "Период:"
+        static let priceTitleText = "Стоимость:"
+        static let dayText = "дней"
+        static let dollarSymbolText = "$"
+        static let editTariffText = "Изменить тариф?"
+        static let basketTitleText = "Корзина"
+        static let buyText = "Купить"
+        static let alertTitleText = "Покупка совершена!"
+        static let oneNumber = 1
+        static let cornerRadiusRectangleNumber: CGFloat = 20
+        static let defaultSpacingNumber: CGFloat = 150
+    }
+    
+    // MARK: - Public Property
```

Comment:
```suggestion
    // MARK: - Body
```
Кидали скрин в важное, как я понял body маскируется отдельно, но могу ошибаться

После боди уже: 
// MARK: - Public Property